pom.xml

```diff
@@ -30,6 +30,11 @@
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
+            <dependency>
```

Comment:
https://github.com/jenkinsci/jackson2-api-plugin can be used instead

Reply:
good catch. fixed